invenio_cli/helpers/docker_helper.py

```diff
@@ -24,6 +25,7 @@ class DockerHelper(object):
     def __init__(self, project_shortname, local=True, log_config=None):
         """Constructor."""
         super().__init__()
+        self.docker_compose = ["docker", "compose"]
```

Comment:
comment: at this point I'm even sceptical of if we need the class attribute `self.docker_compose`, or if we can just replace everywhere `docker-compose` -> `docker compose`.

Not blocking for merging this PR, but something to possibly shelve and explore later as a task to simplify the codebase.